jekyll/_cci2/docker-layer-caching.md

```diff
@@ -7,18 +7,32 @@ order: 5
 
 {% include beta-premium-feature.html%}
 
-If your application is distributed as a Docker image you probably know that the Docker image consists of layers and most of the time only the top layers are being changed. So if your image has many layers you may want to reuse unchanged layers and significantly reduce image build times. By default [Remote Docker Environment]({{ site.baseurl }}/2.0/building-docker-images) doesn't provide caching for layers, but it can be enabled with a special option:
+If your application is distributed as a Docker image, you probably know that this image consists of layers. In addition, it's usually the top layers that end up changing.
+
+If this is the case, you might want to reuse the unchanged layers to significantly reduce image build times. By default, the [Remote Docker Environment]({{ site.baseurl }}/2.0/building-docker-images) doesn't provide layer caching, but you can enable this feature with a special option:
 
 ``` YAML
 - setup_docker_engine:
     reusable: true    # default - false
     exclusive: true   # default - true
 ```
 
-Please note, that this feature requires your project to be whitelisted. Please reach out to your support representative in order to enable it.
+## **`reusable`**
+
+With `reusable: true`, CircleCI will try to reuse your Remote Docker Environment; every layer you built in a previous job will be accessible there. We do our best to reuse the previous environment when we can, but in some cases your job may find itself in a clean environment, even if you've specified `reusable: true`.
+
+If you run many parallel jobs for the same project, and each job requests a reusble environment, all of them will be provided with a Remote Docker Environment. However, not all of them will have cached layers, although this behavior is subject to change over the course of the Beta.
```

Comment:
> not all of them will have cached layers

This's not exactly true, now it sounds like for sure some parallel jobs won't have a cached layer. However they all **might** have it, it just depends on some factors.